backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/button/internal/BpkButtonImpl.kt

```diff
@@ -116,15 +123,6 @@ internal fun ButtonText(text: String, modifier: Modifier = Modifier) {
   )
 }
 
-@Composable
-internal fun ButtonProgress(size: BpkButtonSize, modifier: Modifier = Modifier) {
-  CircularProgressIndicator(
-    modifier = modifier.requiredSize(size.loadingIndicatorSize),
```

Comment:
I think the `loadingIndicatorSize` & `loadingSpinnerColor` can be removed as well